src/api/crosscutRequests.js

```diff
@@ -30,14 +27,16 @@ export const fetchCatchmentJobs = async () => {
         }
         // filter out jobs that aren't site-based
         const siteBasedJobs = resp.jobs.filter((job) => job.algorithm === "site-based")
-        
+
+        // TODO: sort by time it was created
```

Comment:
I think it would be better to sort by the catchment ID, since the catchment ID is how we sort it in the app (and sorting it by catchment ID sorts it by the date created)